lib/content_based.py

```diff
@@ -74,7 +74,7 @@ def get_document_topic_distribution(self):
         return self.document_distribution
 
     @overrides
-    def get_predictions(self):
+    def get_predictions(self, similarity='pearson'):
         """
         Get the expected ratings between users and items.
 
```

Comment:
Add similarity as an option in the json file and handle whether it is one of the 3 or not